screens/ProfilePage.tsx

```diff
@@ -1,55 +1,104 @@
 import * as React from "react";
-import {
-  StyleSheet,
-  Text,
-  View,
-  TextInput,
-  TouchableOpacity,
-} from "react-native";
-
-interface UserInfo {
-  email: string;
-  username: string;
-  fullName: string;
-  phone: string;
-}
+import { useState } from "react";
+import { userContext } from "../App";
+//import the context
+import { useUserContext } from "../App";
+import { StyleSheet, Text, View, Pressable, Alert } from "react-native";
 
 const ProfilePage = () => {
-  const [userInfo, setUserInfo] = React.useState<UserInfo>({
-    email: "",
-    username: "",
-    fullName: "",
-    phone: "",
-  });
+  //extract user info from userContext (aka the user information)
+  const { info } = useUserContext();
+  const onSelling = () => {
+    Alert.alert("Switch to selling");
```

Comment:
I don't feel we need line 12 or line 16 with the alerts. If it's for testing right now that is fine but since the page shows which component is rendered (selling vs wishlist) we don't need the alerts.